src/styles/landing.module.scss

```diff
@@ -0,0 +1,116 @@
+@mixin centerStuff {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin border_outline() {
+  border: none;
+  outline: none;
+}
+@import "./dropdown";
+
+$netflixColor: #e50914;
+$netflixGroundColor: #0d0c0d;
```

Comment:
I saw this variables late only checked you `css` code please move them to variables file